File: src/sdk-reference/cpp/1/collection/list/snippets/list.cpp
try {
  kuzzleio::query_options options;
  options.from = 1;
  options.size = 2;

  std::string collection_list = kuzzle->collection->list("mtp-open-data", &options);

  std::cout << collection_list << std::endl;
  // {"type":"all","collections":[{"name":"pink-taxi","type":"stored"}],"from":1,"size":2}
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/collection/list/snippets/list.cs
try {
  QueryOptions options = new QueryOptions();
  options.from = 1;
  options.size = 2;

  string collection_list = kuzzle.collection.list("mtp-open-data", options);

  Console.WriteLine(collection_list);
  // {"type":"all","collections":[{"name":"pink-taxi","type":"stored"}],"from":1,"size":2}
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.getMessage());
}
